addons/pharma/stringtable.xml

```diff
@@ -3489,6 +3530,7 @@
             <Dutch>Staat toe dat coagulatie kleine wonden beïnvloed tijdens het stollen</Dutch>
             <French>Permettre à la coagulation d'affecter les blessures mineures lors de la coagulation</French>
             <Italian>Consenti alla coagulazione di influenzare le ferite minori durante la coagulazione</Italian>
+            <German>Lassen Sie die Gerinnung zu, um kleinere Wunden zu beeinflussen, wenn es zur Gerinnung kommt</German>
```

Comment:
"Lässt zu, dass Koagulation kleinere Wunden bei Gerinnung beeinflusst" could be better